app/lib/paper_trail_version.rb

```diff
@@ -47,8 +47,8 @@ def shaped_changes
              end
   end
 
-  def marked_urgent?
-    object_changes['urgent_flag']&.last == true
+  def marked_flagged?
+    object_changes['flagged']&.last == true
```

Comment:
This we'll want to keep a vestige of at least for awhile, since we aren't rewriting object history. This should do the trick:

```
```suggestion
    (object_changes['flagged']&.last == true) || (object_changes['urgent_flag']&.last == true)
```
```